composer.json

```diff
@@ -12,7 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
+        "psr/container": "^1.0",
+        "phpunit/phpunit": "5.0.0"
```

Comment:
@stof You're right, it shouldn't be in this PR - added by accident. Will take it out.